Classes/Issues/IssuesViewController.swift

```diff
@@ -642,5 +648,59 @@ final class IssuesViewController: MessageViewController,
         guard let issueType = self.issueType else { return }
         presentLabels(client: client, owner: owner, repo: repo, label: label, type: issueType)
     }
+    
+    // MARK: EditIssueTitleViewControllerDelegate
 
+    func sendEditTitleRequest(newTitle: String, viewController: EditIssueTitleViewController) {
+        let request = V3EditIssueTitleRequest(
+            owner: model.owner,
+            repo: model.repo,
+            issueNumber: model.number,
+            title: newTitle
+        )
+        
+        client.client.send(request) { [weak self] result in
+            switch result {
+            case .success:
+                guard let strongSelf = self else { return }
+                
+                // Update Bookmark with new title
+                if let bookmark = strongSelf.bookmark,
+                    let bookmarkStore = strongSelf.client.bookmarksStore {
+                    IssuesViewController.updateBookmarkTitle(
+                        newTitle: newTitle,
+                        bookmark: bookmark,
+                        bookmarkStore: bookmarkStore
+                    )
+                }
+                
+                // Update issueResult model and timeline
+                if let current = strongSelf.result {
+                    let issueResult = IssuesViewController.updateIssueResultModelTitle(
+                        newTitle: newTitle,
+                        oldTitle: strongSelf.currentIssueTitle ?? "",
+                        username: strongSelf.client.userSession?.username ?? Constants.Strings.unknown,
+                        issueResultModel: current,
+                        width: strongSelf.insetWidth
+                    )
+                    strongSelf.client.cache.set(value: issueResult)
+                }
+                
+            case .failure(let error):
+                Squawk.show(error: error)
+            }
+            
+            viewController.setRightBarItemIdle()
```

Comment:
So the edit VC sets its right bar spinning, but the _issue VC_ is required to set it back? Adds a lot of implicit logic. Another +1 for just putting all this logic inside the edit VC...